Rakefile

```diff
@@ -33,14 +33,13 @@ if cucumber?
   Cucumber::Rake::Task.new(:features) do |t|
     opts = "features --format junit -o #{CUKE_RESULTS} --format pretty -x"
     opts += " --tags #{ENV['TAGS']}" if ENV['TAGS']
-    opts += " --tags ~@skip"
```

Comment:
Why are you removing it?

Reply:
Rolled back with correct tag value

Reply:
Will check again - trying to preserve this tag throws an error.

Reply:
if I remember well the '~' symbol is deprecated
It should be like
```ruby
opts += " --tags not @skip"
```

Reply:
Resolved by @sashaCher